start_docker.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# if requirements.txt has been changed in the last day, automatically rebuild Docker first
+if [[ $(find . -mmin -1440 -type f -print | grep requirements.txt | wc -l) -eq 1 ]]
+  then
```

Comment:
Looks good. Just add a message saying something to the effect that a new requirements file was detected, rebuilding docker image so user knows what's going on.